Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -660,7 +660,7 @@
 workflows[1352] = ['', ['HSCPstop_M_200_13','DIGIUP15','RECOUP15','HARVESTUP15']]
 workflows[1353] = ['', ['RSGravitonToGaGa_13','DIGIUP15','RECOUP15','HARVESTUP15']]
 workflows[1354] = ['', ['WpToENu_M-2000_13','DIGIUP15','RECOUP15','HARVESTUP15']]
-workflows[1355] = ['', ['DisplacedSUSY_stopToBottom_M_300_1000mm_13','DIGIUP15','RECOUP15','HARVESTUP15']]
+workflows[1355] = ['', ['DisplacedSUSY_stopToBottom_M_800_500mm_13','DIGIUP15','RECOUP15','HARVESTUP15']]
```

Comment:
I propose we update only upgrade workflows and leave standard workflow as the old sample since we don't produce this 1355 anymore